spec/GraphQLOverHTTP.md

```diff
@@ -112,72 +178,152 @@ http://example.com/product/graphql
 
 # Serialization Format
 
-The GraphQL specification allows for many [serialization formats to be implemented](https://spec.graphql.org/October2021/#sec-Serialization-Format). Servers and clients over HTTP MUST support JSON and MAY support other, additional serialization formats.
+The GraphQL specification allows for many
+[serialization formats to be implemented](https://spec.graphql.org/draft/#sec-Serialization-Format).
+Servers and clients MUST support JSON and MAY support other, additional
+serialization formats.
+
+For consistency and ease of notation, examples of the response are given in JSON
+throughout this specification.
+
+## Media Types
+
+The following are the officially recognized GraphQL media types to designate
+using the JSON encoding for GraphQL requests and responses over HTTP.
+
+| Name                       | Description                                   |
+| -------------------------- | --------------------------------------------- |
+| `application/graphql+json` | The preferred type; better HTTP compatibility |
+| `application/json`         | To support legacy clients                     |
+
+For details of the shapes of these JSON payloads, please see
+[Request](#sec-Request) and [Response](#sec-Response).
+
+If the media type in a `Content-Type` or `Accept` header includes encoding
+information, then the encoding MUST be `utf-8` (e.g.
+`Content-Type: application/graphql+json; charset=utf-8`). If encoding
+information is not included then `utf-8` should be assumed.
```

Comment:
```suggestion
information is not included then `utf-8` MUST be assumed.
```

I think making this default a strong requirement is essential to ensure interoperability.

Reply:
Agreed